### ISSUE TITLE
[no-member] Instance of 'Item' has no 'save' member

### ISSUE THREAD

**Issue by philbert** [PyCQA/pylint-django#60]
I discovered pylint-django after finding this [stackoverflow thread](http://stackoverflow.com/questions/115977/using-pylint-with-django).

However I still have some false pylint errors working with django:

```
[no-member] Instance of 'Item' has no 'full_clean' member
[no-member] Instance of 'Item' has no 'save' member
```

Is this something that pylint-django can help out with at all? Or is it some peculiar behaviour of django with forms and models that makes hard to deal with?

**Comment by comandrei** [comment]
This seems to be a duplicate of https://github.com/landscapeio/pylint-django/issues/53

**Comment by bittner** [comment]
I have the same problem with `filter`. This is almost certainly a duplicate of #53, and needs to be fixed.

**Comment by atodorov** [comment]
Folks, can somebody post a current reproducer? I'm not convinced I understand the original issue.
